highfidelity/hifi
# some settings cleanup

- adds a description to the Send Data setting... the UI/layout is a little less than desirable.
- removes a setting that was not longer in use (HMD/UI horizontal angular size (degrees))
- moves another developer option into the hidden developer section (Octree/Max Packets per second)
- removes the LOAD DEFAULT SCRIPTS from the General settings, since this option is already on Running Scripts
- removes the Load Scripts from this directory setting, that didn't appear to actually do anything
- moves some obscure items for field of view and tablet in desktop scaling to the Developer/Advanced Settings preferences instead of being front and center

Addresses these recommendations: https://highfidelity.fogbugz.com/f/cases/11624/

# Test Plan:
- Make sure default scripts still work
- Make sure that if you've loaded your own custom scripts, they still work as expected on startup
- Make sure the items noted in fogbugz 11624 are moved or removed
- Make sure the Send Data setting has a description.
